x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-default_policy.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
ℹ️ Because of the way the template registry works module have to put their ILM policies and template into `x-pack/core` resources instead of their own path. This is not ideal but we have to deal with it.

I have created an `entearch` repo there so our changes will be easy to track.